core/PRP/Geometry/plMorphDelta.h

```diff
@@ -20,30 +20,25 @@
 #include "PRP/plCreatable.h"
 #include "Math/hsGeometry3.h"
 
-class HSPLASMA_EXPORT plVertDelta
+struct HSPLASMA_EXPORT plVertDelta
```

Comment:
Why were these changed to structs?

Reply:
It was the simplest way to make all the fields `public`. This seemed to be the convention in libHSPlasma for things that only hold simple data values.

But if you like, I can change it back to `class` and/or wrap the fields in getters/setters. I don't have opinions on the details here, as long as there's *some* way to access the data in the class.